zerotier/default-home-path.patch

```diff
@@ -0,0 +1,11 @@
+--- osdep/OSUtils.dist.cpp	2018-07-12 04:07:34.674510440 +0000
++++ osdep/OSUtils.cpp	2018-07-12 04:08:56.673732991 +0000
+@@ -410,7 +410,7 @@
+ 	return std::string("/var/db/zerotier-one");
+ #else
+ 	// Use /var/lib for Linux and other *nix
+-	return std::string("/var/lib/zerotier-one");
++	return std::string("/hab/svc/zerotier-one/data");
```

Comment:
What impact does this have on configuration plans? Does this prevent a wrapper being created?

Reply:
@themightychris Gentle ping on this question

Reply:
@smacfarlane the upstream project has already closed my issue requesting an environment variable be added for overriding this, so when habitat-sh/habitat#5161 and habitat-sh/habitat#5696 get merged this patch can be removed

I'd be fine with it getting shipped like this for now though, and agree that it may indicate a missing feature